src/index.js

```diff
@@ -1,41 +1,4 @@
-// eslint-disable-next-line import/no-unresolved, import/no-absolute-path
-import './style.css';
-
-// js to access html elements
-const listDiv = document.querySelector('.to-do-list-container');
-
-// array of to do object
-const list = [
-  { description: 'was the dishes-4', completed: false, index: 4 },
-  { description: 'was the dishes-1', completed: false, index: 1 },
-  { description: 'was the dishes-3', completed: false, index: 3 },
-  { description: 'was the dishes-2', completed: false, index: 2 },
-];
-
-// function to display to do list
-const displayList = function display() {
-  // js to sort array of objects by index value
-  list.sort((a, b) => a.index - b.index);
-  // js to iterate through array of object and display sorted array
-  for (let i = 0; i < list.length; i += 1) {
-    const toDoDiv = document.createElement('div');
-    toDoDiv.className = 'to-do';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Dear @Zia123456, I can see you have addressed most of the required changes so far, One thing left (which is one of the main requirements) is, You have commented on a linter rule. As the previous code reviewer mentioned Please, consider removing the linter disabled. Let the linter makes its direct job - find the bugs in the early steps of your project.